conf/messages

```diff
@@ -2681,6 +2690,42 @@ optout.confirmedOptOut.reportQuarterly.reason-list               = This could be
 optout.confirmedOptOut.reportQuarterly.reason1                   = HMRC lowered the income threshold for Making Tax Digital
 optout.confirmedOptOut.reportQuarterly.reason2                   = you reported an increase in your qualifying income in last year’s tax return
 
+optout.confirmedOptOut.scenario1.reporting.obligations.heading                = Your reporting obligations in the future
+optout.confirmedOptOut.scenario1.reporting.obligations.p1                     = You could be required to use Making Tax Digital for Income Tax again in the future if:
+optout.confirmedOptOut.scenario1.reporting.obligations.bullet1                = HMRC lowers the income threshold for it
+optout.confirmedOptOut.scenario1.reporting.obligations.bullet2                = you report an increase in your qualifying income in a tax return
+optout.confirmedOptOut.scenario1.reporting.obligations.inset                  = For example, if your income from self-employment or property, or both, exceeds the £50,000 threshold in the 2024 to 2025 tax year, you would have to use Making Tax Digital for Income Tax from 6 April 2026.
+optout.confirmedOptOut.scenario1.reporting.obligations.p3                     = If this happens, we will write to you to let you know.
+optout.confirmedOptOut.scenario1.reporting.obligations.p4                     = You can check the threshold for qualifying income in the
+optout.confirmedOptOut.scenario1.reporting.obligations.p4.link                = criteria for people who will need to sign up for Making Tax Digital for Income Tax
+
+optout.confirmedOptOut.scenario2.reporting.obligations.heading                = Your reporting obligations from the next tax year onwards
+optout.confirmedOptOut.scenario2.reporting.obligations.inset                  = From 6 April 2025, you will be required to use Making Tax Digital for Income tax.
```

Comment:
This should be dynamic as its refering to the mandated CY+1

Reply:
updated think it should be dynamic now